From fd960fbb3b4c9e29f7109e7963ddf1105911a7a8 Mon Sep 17 00:00:00 2001
From: Mike Lundy <mike@fluffypenguin.org>
Date: Wed, 31 Aug 2011 15:50:06 -0700
Subject: [PATCH 12/17] remove deprecated, unneeded include

libcurl removed this header completely (it hasn't used it for ~7 years)
---
 cloudfsapi.h |    1 -
 1 file changed, 1 deletion(-)

diff --git a/cloudfsapi.h b/cloudfsapi.h
index ff443f7..54a5871 100644
--- a/cloudfsapi.h
+++ b/cloudfsapi.h
@@ -2,7 +2,6 @@
 #define _CLOUDFSAPI_H
 
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 
 #define BUFFER_INITIAL_SIZE 4096
-- 
1.7.9.5

